packages/components/src/components/errorPages/ErrorPage.tsx

```diff
@@ -1,17 +1,60 @@
 import { Button, IconCrossCircle } from 'hds-react';
 import { useRouter } from 'next/router';
-import React, { useContext } from 'react';
+import React from 'react';
 import { Page as RHHCPage } from 'react-helsinki-headless-cms';
 import { MAIN_CONTENT_ID } from '../../constants';
 import useErrorsTranslation from '../../hooks/useErrorsTranslation';
 import useLocale from '../../hooks/useLocale';
-import NavigationContext from '../../navigationProvider/NavigationContext';
+import type { Language, LanguageCodeEnum, Menu } from '../../types';
```

Comment:
importing from index.ts sometimes causes infinite loop problems, I would rather prefer to import these separately from their own file itself.

Reply:
actually, they are all wrong (and still the same). They should be types from the HCRC-lib, not any internal ones. Coincidence that they are now exactly the same.

I'll change them to 

```typescript
import type {
  Language,
  LanguageCodeEnum,
  Menu,
} from 'react-helsinki-headless-cms';
import { Page as RHHCPage } from 'react-helsinki-headless-cms';
```